components/brave_wallet/brave_wallet_constants.h

```diff
@@ -6,7 +6,7 @@
 #ifndef BRAVE_COMPONENTS_BRAVE_WALLET_BRAVE_WALLET_CONSTANTS_H_
 #define BRAVE_COMPONENTS_BRAVE_WALLET_BRAVE_WALLET_CONSTANTS_H_
 
-enum class BraveWalletWeb3ProviderTypes { ASK, NONE, CRYPTO_WALLETS, METAMASK };
+enum class BraveWalletWeb3ProviderTypes { NONE, CRYPTO_WALLETS, METAMASK };
```

Comment:
I think we're going to have to rename `ASK` to `ASK_OBSOLETE`. Otherwise people that had MetaMask will start to have Crypto Wallets since this is stored as an int in prefs

Reply:
Done